.github/workflows/r-ci.yml

```diff
@@ -52,6 +53,63 @@ jobs:
       - name: Bootstrap
         run: cd apis/r && tools/r-ci.sh bootstrap
 
+      - name: Set additional repositories (macOS)
+        if: ${{ matrix.os != 'ubuntu-latest' }}
+        run: echo 'options(repos = c("https://tiledb-inc.r-universe.dev", getOption("repos")))' | tee -a ~/.Rprofile
+
+      - name: Set additional repositories (Linux)
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        run: |
+          rversion <- paste(strsplit(as.character(getRversion()), split = '\\.')[[1L]][1:2], collapse = '.')
+          codename <-  system('. /etc/os-release; echo ${VERSION_CODENAME}', intern = TRUE)
+          repo <- "https://tiledb-inc.r-universe.dev"
+          (opt <- sprintf('options(repos = c("%s/bin/linux/%s/%s", "%s", getOption("repos")))', repo, codename, rversion, repo))
+          cat(opt, "\n", file = "~/.Rprofile", append = TRUE)
```

Comment:
This won't work, as I just discovered.  r-universe already jumped to noble whereas ubuntu-latest will be jammy for some time.  

![image](https://github.com/single-cell-data/TileDB-SOMA/assets/673121/ff9b30fe-c97c-4989-9a9e-bb83fe28d676)

So we are once again between a rock and a hard place.  Once r2u jumps to 0.27.0 (by default tomorrow we're in trouble and the above won't save us.  :disappointed: